dopamine/replay_memory/circular_replay_buffer.py

```diff
@@ -135,6 +136,14 @@ def __init__(self,
       reward_dtype: np.dtype, type of elements in the reward.
       lock: lock object to use for protection against concurrent access. If
         `None` then locking is disabled.
+      max_trajectory_buffer: int, the maximum size of the trajectory contained
```

Comment:
After having read through more code, I don't think that the name is quite accurate.  This is not just the size of the buffer -- it in effect defines the maximum size of a trajectory.

Reply:
Renamed to `max_trajectory_size`. Good point.

Reply:
renamed to `contiguous_trajectories`.